presto-hive/src/main/java/io/prestosql/plugin/hive/HiveWriterFactory.java

```diff
@@ -565,6 +586,16 @@ else if (insertExistingPartitionsBehavior == InsertExistingPartitionsBehavior.ER
                 hiveWriterStats);
     }
 
+    private static Path createHiveBucketPath(Path subdirPath, int bucketToUse, Map<String, String> tableParameters)
+    {
+        if (isInsertOnlyTable(tableParameters)) {
```

Comment:
We could write this so it's easier to see the difference between the two cases
```java
String nameFormat = isInsertOnlyTable(tableParameters) ? "%05d_0" : "bucket_%05d";
return new Path(subdirPath, String.format(nameFormat, bucketToUse));

Reply:
Nicer; changed.